Makefile

```diff
@@ -100,6 +100,21 @@ PROD_TEST_CMD := $(TEST_CMD) \
 run-locust-tests: 
```

Comment:
Maybe rename these to remove the word locust and instead using a term which describes what is being tested?

`run-quota-tests`

etc - same for the others